src/screens/AdminPortal/OrgList/OrgList.spec.tsx

```
@@ -2743,3 +2868,172 @@ describe('Email Verification Actions Tests', () => {
+  mockPaginationFactory.MockComponent = ({
+    onRowsPerPageChange,
+    count,
+  }: {
+    onRowsPerPageChange: (event: { target: { value: string } }) => void;
+    count: number;
+  }) => (
+    <div data-testid="mock-pagination">
+      <button
+        data-testid="set-rows-0"
+        onClick={() => onRowsPerPageChange({ target: { value: '0' } })}
+      >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)